pg_backend_support.c

```diff
@@ -316,6 +316,7 @@ expand_dynamic_library_name(const char *name)
 
 	have_slash = (strchr(name, '/') != NULL);
 
+#ifndef WIN32
```

Comment:
It tries to guess SO/DLL file name starting with no file name extension. Apparently it will fail and throwing error will terminate the backend if built in debug mode.